pkg/services/dashboardsnapshots/service/service.go

```diff
@@ -4,27 +4,39 @@ import (
 	"context"
 
 	"github.com/grafana/grafana/pkg/components/simplejson"
+	"github.com/grafana/grafana/pkg/services/dashboards"
 	"github.com/grafana/grafana/pkg/services/dashboardsnapshots"
 	"github.com/grafana/grafana/pkg/services/secrets"
 )
 
 type ServiceImpl struct {
-	store          dashboardsnapshots.Store
-	secretsService secrets.Service
+	store            dashboardsnapshots.Store
+	secretsService   secrets.Service
+	dashboardService dashboards.DashboardService
 }
 
 // ServiceImpl implements the dashboardsnapshots Service interface
 var _ dashboardsnapshots.Service = (*ServiceImpl)(nil)
 
-func ProvideService(store dashboardsnapshots.Store, secretsService secrets.Service) *ServiceImpl {
+func ProvideService(store dashboardsnapshots.Store, secretsService secrets.Service, dashboardService dashboards.DashboardService) *ServiceImpl {
 	s := &ServiceImpl{
-		store:          store,
-		secretsService: secretsService,
+		store:            store,
+		secretsService:   secretsService,
+		dashboardService: dashboardService,
 	}
 
 	return s
 }
 
+func (s *ServiceImpl) ValidateDashboardExists(ctx context.Context, orgId int64, dashboardUid string) (bool, error) {
+	_, err := s.dashboardService.GetDashboard(ctx, &dashboards.GetDashboardQuery{UID: dashboardUid, OrgID: orgId})
+	if err != nil {
+		return false, err
+	}
+
+	return true, nil
```

Comment:
Isn't there a better way? I don't like theses sentences when talking about boolean values.
I mean, we can do something like 
```suggestion
	return err == nil, err
```
Maybe it's not more readable?

Reply:
So we decided to not return any bool value and just the error since it will always throw one if dashboardNotFound